docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1334,6 +1334,19 @@ Query Parameters:
 
 -   **Content-type** – Set to `"application/tar"`.
 -   **X-Registry-Config** – base64-encoded ConfigFile object
+-   **X-Docker-BuildArgs** – base64-encoded JSON map of string pairs for build-time
```

Comment:
We should be consistent though - unless we're going to move all of 'em to query params I'm not sure it makes sense to just move this one.